demos/python_demos/rag_chatbot/README.md

```diff
@@ -26,18 +26,52 @@ Building the image with all required python dependencies is required. Follow the
 ```bash
 git clone https://github.com/openvinotoolkit/model_server.git
 cd model_server
-make python_image BASE_OS=redhat OVMS_CPP_DOCKER_IMAGE=registry.connect.redhat.com/intel/openvino-model-server OVMS_CPP_IMAGE_TAG=2024.0
+make python_image BASE_OS=redhat OVMS_CPP_DOCKER_IMAGE=registry.connect.redhat.com/intel/openvino-model-server OVMS_CPP_IMAGE_TAG=2024.1
 ```
 It will create an image called `registry.connect.redhat.com/intel/openvino-model-server:py`
 
 You can also build Ubuntu 22.04 image:
 ```
-make python_image BASE_OS=ubuntu OVMS_CPP_DOCKER_IMAGE=openvino/model_server OVMS_CPP_IMAGE_TAG=2024.0
+make python_image BASE_OS=ubuntu OVMS_CPP_DOCKER_IMAGE=openvino/model_server OVMS_CPP_IMAGE_TAG=2024.1
 ```
 It will create an image called `openvino/model_server:py`
 
+## OpenVINO Model Server deployment with online models pulling from Hugging Face Hub
 
-## Download LLM model
+In this demo, OpenVINO Model Server has an option to pull the required models from Hugging Face Hub.
+It's a simple deployment option because it doesn't require models preparation which have to be attached to the container. Just the demo scripts and configation files are required in the container at startup.
+
+What needs to be prepared is a list of documents, which should give context of RAG analysis. It is provided in a format of text file containing URLs of the documents sources:
+
+```bash
+echo "https://gist.githubusercontent.com/ryanloney/42b8ebe29f95ebd4382ee0b2bb50bea2/raw/cfbb679fefb6babec675c7806254a5fff29a5e6b/aipc.txt" > demos/python_demos/rag_chatbot/servable_stream/docs.txt
+```
+Now the model server can be started:
+```bash
+docker run -d -v ${PWD}/demos/python_demos/rag_chatbot/servable_stream:/config -p 9000:9000 -p 8000:8000 \
+-e SELECTED_MODEL=${SELECTED_MODEL} \
+registry.connect.redhat.com/intel/openvino-model-server:py --config_path /config/config.json --port 9000 --rest_port 8000
+```
+> **NOTE** The model loading time for the sequential attempts can be improved by mounting the Hugging Faces cache in the docker command:
+`-v ${HOME}/.cache:/hf-cache -e HF_HOME=/hf-cache/huggingface`.
+> **NOTE** For model llama-2-chat-7b, which require autorization in Hugging Faces hub, it might be needed to pass also HF_TOKEN environment variable.
```

Comment:
```suggestion
> **NOTE** For model llama-2-chat-7b, which require authorization in Hugging Faces hub, it might be needed to pass also HF_TOKEN environment variable.
```